src/www/css/app.css

```diff
@@ -784,10 +800,6 @@ video {
   height: 0.75rem;
 }
 
-.h-32 {
```

Comment:
Can I just check that these are expected deletions?

Reply:
Here as well.